pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -235,21 +235,29 @@ func (r *Retriever) readConfigMap(cr *unstructured.Unstructured, name string, it
 		// update cache after reading configmap/secret in cache
 		r.cache[fromPath].(map[string]interface{})[path].(map[string]interface{})[k] = value
 		// making sure key name has a configMap reference
-		r.store(cr, fmt.Sprintf("configMap_%s", k), []byte(value))
+		r.store(envPrefix, cr, fmt.Sprintf("configMap_%s", k), []byte(value))
 	}
 
 	r.Objects = append(r.Objects, u)
 	return nil
 }
 
 // store key and value, formatting key to look like an environment variable.
-func (r *Retriever) store(u *unstructured.Unstructured, key string, value []byte) {
+func (r *Retriever) store(envPrefix string, u *unstructured.Unstructured, key string, value []byte) {
```

Comment:
I wonder if it's better to have `envPrefix *string` ? There's a chance that if the `envVarPrefix` was skipped by the user in the CR, the value would actually resolve to an empty string.